### PR TITLE
Feature/dashboard layout

### PR DESCRIPTION
**Pull request by kajalelohai**
- A page split in 2 parts: Sidebar and Main
- "Send Feedback" button at bottom of Sidebar
- Sidebar header with Organization logo, user profile and context menu icon (no context menu in this ticket)

### CODE REVIEW

**Comment by bitspook** [comment]
Please attach screenshots with UI related PRs

File: ui/public/assets/images/vector.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="4" viewBox="0 0 12 4" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (bitspook) [inline ui/public/assets/images/vector.svg]: Please choose a better name for this image. `vector.svg` reveals nothing about its conents.

File: ui/src/pages/dashboard/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        border-radius: 50%;
+        width: 8px;
+        height: 8px;
+        background: $color-online;
+        border: 1px solid $color-bg-dark;
+        position: absolute;
+        align-self: flex-end;
+        margin-top: 1px;
+        margin-left: 1px;
+      }
```
Comment (bitspook) [inline ui/src/pages/dashboard/index.module.scss]: I recommend not to unnecessarily nest classes. Until it's needed, avoid nesting styles.

File: ui/src/pages/dashboard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import HEAD from 'next/head';
+import Link from 'next/link';
+import c from './index.module.scss';
+
+const Dashboard = () => {
+  return (
+    <>
+      <HEAD>
+        <title>SOS - Dashboard</title>
```
Comment (bitspook) [inline ui/src/pages/dashboard/index.tsx]: I think we are naming it `SoS` in rest of the app (small `o`). Please follow the same convention.

**Review by bitspook — CHANGES_REQUESTED** [review]

File: ui/src/pages/dashboard/index.module.scss (hunk 2 above)
Comment (kajalelohai) [reply ui/src/pages/dashboard/index.module.scss]: Shall I change this or its for the further application?

**Review by bitspook — APPROVED** [review]

**Pull request merged**